cypress/integration/search/search.spec.js

```diff
@@ -260,116 +260,4 @@ describe('Search', () => {
       })
     }
   )
-
-  context('When the role of user allows searching by work order only', () => {
```

Comment:
Why are all these tests being removed?

Reply:
I have re-added the tests, except for the test that 'shows an error when property search data is entered'